syft/pkg/cataloger/javascript/package.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/anchore/syft/syft/source"
 )
 
+const aliasSeparatorPackageLockV1 = "npm:"
```

Comment:
the smallest of nits: I think this would be a `prefix` rather than a `separator`, no?

Reply:
nice spot: updated to `prefix`